### DISCUSSION TITLE
[ 求助 ] 如何合理训练邦布

### DISCUSSION THREAD

**Discussion by haxyyx** [share121/inter-knot#3142] [其他]
![委托](https://github.com/user-attachments/assets/2c3df458-4131-4ed5-a924-d11adb8a2975)

今天在训练邦布，准备让邦布去打邦布对决，结果邦布不愿意离家出走了，我每天才让它训练10小时，别人的专业对战邦布一天根本不断的训练为什么就不能跟那些邦布学一学，好吧说那么多也没用，如果我不改变训练方法，我的小邦布应该是不回来了，所以绳网的大家有什么邦布对战训练的方法吗，迟早有一天我会和我的邦布站在最巅峰！

**Comment by I-ikun** [comment]
建议给我

**Reply by haxyyx** [reply]
喂！虽然离家出走了，也不是你这样挖墙脚的吧，还有拐卖，偷窃邦布是违法的！
